### ISSUE TITLE
Data navigation - Should be able to use TAB key to navigate data/now viewing selections

### ISSUE THREAD

**Issue by Layoric** [TerriaJS/terriajs#507]
A user wanting to turn data layers on and off should be able to do so using keyboard navigation. At the moment, no input/label or tabindexs are used for the left hand menu (except the search box).

![image](https://cloud.githubusercontent.com/assets/234642/5995667/795b2c4e-aaed-11e4-8733-68a5f08c68d8.png)

**Comment by kring** [comment]
Thanks @Layoric, we will definitely do this as well sometime soon.  As you know, support for keyboard support in National Map is not great at the moment.

**Comment by hilarycinis** [comment]
Agreed, thanks as this would be part of accessibility options too.

**Comment by Layoric** [comment]
Just as an update to this, a user at GA discovered the following behaviour when trying to use tab to navigate the current version of NationalMap.

![image](https://cloud.githubusercontent.com/assets/234642/7804758/8a9b037e-03aa-11e5-95f7-177c52b1bb9c.png)

Screenshot above is behaviour in Chrome 42, but behaviour slightly differs depending on browser. Problem occurs when search field grabs focus and search tab having not been clicked.

**Comment by kring** [comment]
Thanks @Layoric.  Keith submitted a fix for the immediate problem above in #717.  Proper keyboard navigation will have to wait a bit longer.

**Comment by chloeleichen** [comment]
This issue is related to #959
